tests/test_claude_wrapper_tmux_env_regression.py

```
@@ -0,0 +1,188 @@
+            env["CMUX_CLAUDE_WRAPPER_DEBUG_LOG"] = str(debug_log)
+        else:
+            env.pop("CMUX_CLAUDE_WRAPPER_DEBUG", None)
+            env.pop("CMUX_CLAUDE_WRAPPER_DEBUG_LOG", None)
+
+        for key, value in env_overrides.items():
+            if value is None:
+                env.pop(key, None)
+            else:
+                env[key] = value
+
+        proc = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)